### DISCUSSION TITLE
ui top dashboard for apm metrics

### DISCUSSION THREAD

**Discussion by houbinglei** [apache/skywalking#9941] [Q&A] [answered]
i have used skywalking 9.x, but i cant found the  services load panel sorted like this:
![image](https://user-images.githubusercontent.com/37292926/201016569-d9c2e581-2a7b-4167-b054-d2fd1f06d24e.png)

how can i make a panel like this ? for example : list the top ten on service load?

**Comment by wu-sheng** [comment, marked as answer]
No, there is no global concept any more. 
Based on community feedback, listing services by response time order doesn't make any sense.
Some services are slower because they process more logic, that is all.

**Reply by houbinglei** [reply]
thanks ! i see!
